feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle/configurator-attribute-single-selection-bundle.component.html

```diff
@@ -19,7 +20,6 @@
 
   <cx-configurator-attribute-product-card
     id="{{ createAttributeValueIdForConfigurator(attribute, value.valueCode) }}"
-    (handleDeselect)="onDeselect()"
```

Comment:
deselecting shall be still possible for non mandatory fields, so i believe will still require it.

Reply:
fixed